UEBPrep/app/models/user.rb

```diff
@@ -12,6 +12,14 @@ class User < ActiveRecord::Base
     doc.api_key = doc.generate_api_key
   end
 
+  has_attached_file :avatar,
+                    styles: { small: "64x64", med: "100x100", large: "200x200" }
```

Comment:
TODO: Figure out what the ideal sizes are for the three main screen sizes: phones, tablet, desktop.